dll/win32/shell32/CDefView.cpp

```diff
@@ -402,6 +402,7 @@ HRESULT WINAPI CDefView::Initialize(IShellFolder *shellFolder)
 {
     m_pSFParent = shellFolder;
     shellFolder->QueryInterface(IID_PPV_ARG(IShellFolder2, &m_pSF2Parent));
+
```

Comment:
What does this achieve? :P

Reply:
It negates the removal of this line in an earlier commit.